privateGPT.py

```diff
@@ -23,6 +25,17 @@
 def main():
     # Parse the command line arguments
     args = parse_arguments()
+
+    # Setup logging if enabled
+    if args.log_level is not None:
+        file_handler = logging.FileHandler(
+            filename=f'{os.path.basename(__file__)}.log')
+        stdout_handler = logging.StreamHandler(sys.stdout)
+        handlers = [file_handler, stdout_handler]
+        logging.basicConfig(format='%(asctime)s %(levelname)-8s %(message)s',
+                            handlers=handlers, level=args.log_level, force=True)
+
```

Comment:
Maybe move it to the `logs` folder?

```python
# Setup logging if enabled
if args.log_level is not None:
    log_folder = "logs"  # Specify the folder where you want to store the log files
    os.makedirs(log_folder, exist_ok=True)  # Create the log folder if it doesn't exist

    log_file = os.path.join(log_folder, f'{os.path.basename(__file__)}.log')  # Path to the log file
    file_handler = logging.FileHandler(filename=log_file)
    stdout_handler = logging.StreamHandler(sys.stdout)
    handlers = [file_handler, stdout_handler]
    logging.basicConfig(format='%(asctime)s %(levelname)-8s %(message)s', handlers=handlers, level=args.log_level, force=True)
```

Reply:
Change my PR to move the logs to a folder called 'logs'